### DISCUSSION TITLE
suggestion: add `touch` command

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/terminal#14500] [Ideas]
`touch` command - creates empty file/s

**vvv realisation code example below vvv**

![touch-image](https://user-images.githubusercontent.com/101216311/206029309-409a4588-6739-47e6-9347-b1b4e1036004.jpeg)

**Comment by jameswylde** [comment]
In terms of scope, this suggestion is perhaps more likely better suited for the Powershell [repo](https://github.com/PowerShell/PowerShell) rather than this Terminal discussion.

However, you can always add a function replicating _touch_ to your [Powershell profile](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-7.3) in the mean time - the below from [Ansgar Wiechers](https://stackoverflow.com/a/32451953) works fine enough:

```
function touch {
  Param(
    [Parameter(Mandatory=$true)]
    [string]$Path
  )

  if (Test-Path -LiteralPath $Path) {
    (Get-Item -Path $Path).LastWriteTime = Get-Date
  } else {
    New-Item -Type File -Path $Path
  }
}
```

**Comment by ghost** [comment]
discussion is outdated
